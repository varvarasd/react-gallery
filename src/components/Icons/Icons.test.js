import React from 'react';
import { render } from '@testing-library/react';
import { LeftArrow, RightArrow } from '.';

describe('LeftArrow', () => {
    it('renders with given width', () => {
        const { container } = render(<LeftArrow width={'10px'}/>);
        expect(container.firstChild).toMatchSnapshot();
    });
});

describe('RightArrow', () => {
    it('renders with given width', () => {
        const { container } = render(<RightArrow width={'10px'}/>);
        expect(container.firstChild).toMatchSnapshot();
    });
});