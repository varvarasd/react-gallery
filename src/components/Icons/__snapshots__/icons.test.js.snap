// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`LeftArrow renders with given width 1`] = `
<svg
  viewBox="0 0 50 80"
  width="10px"
>
  <polyline
    fill="none"
    points="45.63,75.8 0.375,38.087 45.63,0.375 "
    stroke="#808080"
    stroke-linecap="round"
    stroke-linejoin="round"
    stroke-width="5"
  />
</svg>
`;

exports[`RightArrow renders with given width 1`] = `
<svg
  viewBox="0 0 50 80"
  width="10px"
>
  <polyline
    fill="none"
    points="0.375,0.375 45.63,38.087 0.375,75.8 "
    stroke="#808080"
    stroke-linecap="round"
    stroke-linejoin="round"
    stroke-width="5"
  />
</svg>
`;
